liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_metadata_columns-001.sql

```diff
@@ -1,8 +1,8 @@
 IF NOT EXISTS (SELECT 1
                FROM sysobjects
-               WHERE name = 'nrt_interview_columns'
+               WHERE name = 'nrt_metadata_columns'
                  and xtype = 'U')
-CREATE TABLE dbo.nrt_interview_columns
+CREATE TABLE dbo.nrt_metadata_columns
 (
```

Comment:
I am not able to comment on the columns that haven't been changed. Are NEW_FLAG, LAST_CHG_TIME, LAST_CHG_USER_ID nullable? If so, let's specify that.

Reply:
Thank you for pointing that out, fixed